plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -701,20 +664,21 @@ def get_deploy_transaction(
             if kwargs.get("chain_id", None) is not None
             else self.chain_id
         )
-        account_number, sequence = self._try_get_account_number_and_sequence(
-            deployer_address
-        )
+
+        account_number = kwargs.pop("account_number", None)
+        sequence = kwargs.pop("sequence", None)
```

Comment:
I looked it up and we use this method a lot without providing `account_number` and `sequence`. With this change, these fields must be known beforehand. How would that be known? What would the interaction look like differently?

Reply:
Why did you add the option of providing the fields as inputs to this method?

Reply:
In real life, this will be for an offline tx generation but now it is used for tests so we don't have to refill balance from the faucet in order to generate messages - an account without balance has no account number nor sequence and getting it will fail.